proto/rootfs.proto

```diff
@@ -0,0 +1,54 @@
+syntax = "proto3";
```

Comment:
Building an API for rootfs uploads is further out-of-scope than I would recommend for this challenge.

Reply:
Understood. I'll remove it when I update the PR.

Reply:
Removed rootfs api.